src/faebryk/libs/picker/picker.py

```diff
@@ -29,13 +31,21 @@ class Part:
     supplier: Supplier
 
 
+@dataclass
+class PartIdentifier:
+    manufacturer: str
+    partno: str
+    datasheet: str
+
+
```

Comment:
remove, just use info

Reply:
If you find it important to have the strings defined somewhere instead of using str everywhere you can add a StrEnum for the descriptive properties.

```python
class DescriptiveProperties(StrEnum):
    manufacturer = "Manufacturer"
    partno = "Partnumber"
    ...
```

Then you can just do in the pickeroptions:
```python
   ...
   info = {DescriptiveProperties.manufacturer: "ABC", DescriptiveProperties.partno: "123"}
   ...
```